reflex/app_module_for_backend.py

```diff
@@ -4,12 +4,14 @@
 from concurrent.futures import ThreadPoolExecutor
 
 from reflex import constants
+from reflex.utils import telemetry
 from reflex.utils.exec import is_prod_mode
 from reflex.utils.prerequisites import get_app
 
 if "app" != constants.CompileVars.APP:
     raise AssertionError("unexpected variable name for 'app'")
 
+telemetry.send("reload")
```

Comment:
i think we should call this a `"compile"` event rather than `"reload"` because this will get hit for run dev, run prod, hot reload, and export